tests/wpt/meta/content-security-policy/securitypolicyviolation/targeting.html.ini

```diff
@@ -8,6 +8,3 @@
 
   [Correct targeting inside shadow tree (inline handler).]
```

Comment:
We get a couple of steps closer in this test. Now it correctly passes blocks 1 to 3. It then fails on the shadow tree inline handler case, where it doesn't correctly compute the shadow root element. Not sure why this fails yet, since it doesn't appear to be firing an event. But leaving that up to another investigation, as I want to make slow progress on at least the parts that do work.